string arrived
int32 objOnTable
---
int32 go
int32 req_objs
